content/MBC2.md

```diff
@@ -18,13 +18,18 @@ battery to save data during power-off though. As the data consists of
 4bit values, only the lower 4 bits of the "bytes" in this memory area
 are used.
 
+### A200-BFFF - 15 "echoes" of A000-A1FF
+Only the bottom 9 bits of the address are used to index into the internal
+RAM, so RAM access repeats.
+
 ## Control Registers
 
 ### 0000-3FFF - RAM Enable and ROM Bank Number (Write Only)
 
 This address range is responsible for both enabling/disabling the RAM
-and for controlling the ROM bank number. Bit 8 of the address determines
-whether to control the RAM-enable flag or the ROM bank number.
+and for controlling the ROM bank number. Bit 8 of the address (the least
+significant bit of the upper address byte) determines whether to control
+the RAM-enable flag or the ROM bank number.
```

Comment:
I don't like the inconsistency between "RAM-enable" and "ROM bank number", maybe remove the hyphen?